tools/ola_trigger/contrib/elgato_key_light.conf

```diff
@@ -0,0 +1,127 @@
+# OLA Trigger config to control Elgato Key Light or Key Light Airs
+# Copyright (C) 2020-2023 Johan Nilsson. https://gobo.ws
+
+
+
+# DMX chart
+
+###############################################################################
+
+# +---------+---------+----------------------------+
+# | Channel | Value   | Function                   |
+# +---------+---------+----------------------------+
+# | 1       | 0-100   | Dimmer (0-100%)            |
+# |         | 101-255 | Unused range               |
+# +---------+---------+----------------------------+
+# | 2       |         | Color temperature (kelvin) |
+# |         | 0-3     | Unused range               |
```

Comment:
> What's the reason for missing this bit of the range, just to make the numbers nicer?

I wrote the original script some years ago and I think that my idea was that if you set the fader on that channel to 0 then the color temperature should not be changed, but we can definitely start at 0 instead of 4.

Reply:
Ah, yeah I'd not considered wanting to leave it as is. That feels like a bad behaviour as a DMX fixture, but perhaps a good workaround if you're trying to control it with a simpler lighting desk.